oe_paragraphs.post_update.php

```diff
@@ -129,3 +129,70 @@ function oe_paragraphs_post_update_10004(array &$sandbox): void {
     $display->save();
   }
 }
+
+/**
+ * Installs Facts and figures and Fact paragraphs.
+ */
+function oe_paragraphs_post_update_10005(array &$sandbox) {
+  // If paragraph already exists, we bail out.
+  $paragraph = \Drupal::entityTypeManager()->getStorage('paragraphs_type')->load('oe_facts_figures');
+  if ($paragraph) {
+    return t('Facts and figures paragraph exists, no action required.');
+  }
+
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs') . '/config/post_updates/10005');
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraphs_type');
+
+  // Create the paragraphs.
+  $paragraphs = [
+    'paragraphs.paragraphs_type.oe_fact',
+    'paragraphs.paragraphs_type.oe_facts_figures',
+  ];
+  foreach ($paragraphs as $paragraph) {
+    $paragraph_storage->create($storage->read($paragraph))
+      ->save();
+  }
+
+  // Configure fields and entity form and view displays for both paragraphs.
+  $config_ids = [
+    'field.storage.paragraph.field_oe_subtitle',
+    'field.field.paragraph.oe_fact.field_oe_icon',
+    'field.field.paragraph.oe_fact.field_oe_plain_text_long',
+    'field.field.paragraph.oe_fact.field_oe_subtitle',
+    'field.field.paragraph.oe_fact.field_oe_title',
+    'field.field.paragraph.oe_facts_figures.field_oe_link',
+    'field.field.paragraph.oe_facts_figures.field_oe_paragraphs',
+    'field.field.paragraph.oe_facts_figures.field_oe_title',
+    'core.entity_form_display.paragraph.oe_fact.default',
+    'core.entity_form_display.paragraph.oe_facts_figures.default',
+    'core.entity_view_display.paragraph.oe_fact.default',
+    'core.entity_view_display.paragraph.oe_facts_figures.default',
+  ];
+
+  $config_manager = \Drupal::service('config.manager');
+  $entity_manager = \Drupal::entityTypeManager();
+  foreach ($config_ids as $config_id) {
+    $config_record = $storage->read($config_id);
+    $entity_type = $config_manager->getEntityTypeIdByName($config_id);
+    $entity_storage = $entity_manager->getStorage($entity_type);
+    $entity = $entity_storage->createFromStorageRecord($config_record);
+    $entity->save();
+  }
+
+  return t('Facts and figures and Fact paragraphs have been installed.');
+}
+
+/**
+ * Marks unsupported ECL icons as deprecated.
+ */
+function oe_paragraphs_post_update_10006(array &$sandbox): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs') . '/config/post_updates/10006');
+  $config_manager = \Drupal::service('config.manager');
+  $entity_manager = \Drupal::entityTypeManager();
+  $config_record = $storage->read('field.storage.paragraph.field_oe_icon');
+  $entity_type = $config_manager->getEntityTypeIdByName('field.storage.paragraph.field_oe_icon');
+  $entity_storage = $entity_manager->getStorage($entity_type);
+  $entity = $entity_storage->load($config_record['id']);
+  $entity_storage->updateFromStorageRecord($entity, $config_record);
+  $entity->save();
```

Comment:
Let's just change those 3 values in the field configuration so we don't have to maintain a full copy of it in the 10006 directory.